### DISCUSSION TITLE
Birdseye view support for audio stream

### DISCUSSION THREAD

**Discussion by chrislawso** [blakeblackshear/frigate#11430] [Ideas]
Is there any method in the configuration file to enable audio streaming for all or certain cameras to play audio stream in birdseye view? 

In order to get audio stream of certain cameras I keep additional browser windows open of individual camera streams where I can enable audio and adjust volume and in separate browser window I open birdseye view. This is not ideal just to be able to get audio streams. 

Ideally I would like to be able to only have the birdseye view page open to conserve bandwidth and be able to get certain cameras audio playing in birdseye view. Ideally the ability to configure which cameras to play audio, ie several cameras simultaneously playing audio, and also adjustable volume levels per camera in birdseye view would be great.

Thanks

**Comment by bourbonbenny** [comment]
Also interested in this feature. Have been looking through documentation and message boards but don't see anything to accomplish this. 

My use case would be viewing on mobile on home assistant app.

**Comment by LucaZiegler** [comment]
I wonder why it's still not possible

**Comment by hawkeye217** [comment]
Frigate does not process and decode audio for the `detect` stream from your cameras, which birdseye uses. For audio on your live streams, use the [enhanced live view](https://docs.frigate.video/configuration/live/) available in Frigate 0.14.

**Comment by LucaZiegler** [comment]
![Screenshot_20240918_172712_Home Assistant~2](https://github.com/user-attachments/assets/188bb7f2-ad00-4208-b9ce-6e83f4e49d3d)

Thank you it's working!
